src/graphql/Tokens/api-keys.js

```diff
@@ -0,0 +1,33 @@
+import gql from 'graphql-tag'
+
+export default function(isCloud) {
+  return gql`
+    query ApiKeys(
+      $limit: Int
+    ) {
+    auth_api_key(
+        limit: $limit
+        order_by: { created: desc }
+      ) {
+
+        id
+        name
+        created
+        expires_at
+        default_tenant_id
+        user_id
+        updated
+
+      ${
+        isCloud
+          ? `
```

Comment:
I don't think we need the isCloud check here as this query will only be run by cloud.

Reply:
I'd revert to using the .gql query but add ```created_by {id, username}```